juju/juju
# juju-run is now juju-exec

This PR updates the "juju-run" infrastructure to match the CLI changes where "juju run" is now "juju exec".

The /usr/bin/juju-run symlink is now /usr/bun/juju-exec, and any legacy /usr/bin/juju-run symlink is removed.

## QA steps

Deploy a unit on 2.9
upgrade to 3 and ensure /usr/bin/juju-run symlink is removed
juju exec --unit foo/0 hostname
juju operations --name juju-exec
juju ssh foo/0 "sudo juju-exec pwd"
juju ssh foo/0 "sudo juju-exec config-get"
juju exec --unit foo/0 "sudo juju-exec  pwd" -> error
